src/middlewared/middlewared/api/base/types/iscsi.py

```diff
@@ -0,0 +1,5 @@
+from typing import Literal, TypeAlias
+
+__all__ = ["IscsiAuthType"]
+
+IscsiAuthType: TypeAlias = Literal['NONE', 'CHAP', 'CHAP_MUTUAL']
```

Comment:
Is this being used?

Reply:
Not yet, but the plan is that it will be.